tests/tensor_search/test_create_index.py

```diff
@@ -358,9 +356,8 @@ def test_configurable_max_number_of_replicas(self):
         maximum_number_of_replicas = 5
         large_intended_replicas_count = 10
         small_intended_replicas_count = 3
-        from marqo.tensor_search.models.settings_object import settings_schema
 
-        with patch.dict(settings_schema["properties"][NsField.number_of_replicas], maximum=maximum_number_of_replicas):
+        with patch.dict(os.environ, {EnvVars.MARQO_MAX_NUMBER_OF_REPLICAS: str(maximum_number_of_replicas)}):
```

Comment:
We should also make another test changing `EnvVars.MARQO_EF_CONSTRUCTION_MAX_VALUE`. Then another one changing both.

Reply:
was this addressed?